.github/workflows/test.yaml

```diff
@@ -24,6 +24,10 @@ jobs:
           - dev
           - release
 
+        target_triple:
+          - x86_64-pc-windows-msvc
+          # - aarch64-pc-windows-msvc FIXME: Add support for executing ARM64 tests. Maybe use target specific test runner in emulator: https://doc.rust-lang.org/cargo/reference/config.html#target
```

Comment:
Support for ARM64 tests is possibly possible in the future via target-specific test runner: https://doc.rust-lang.org/cargo/reference/config.html#target